openvdb/openvdb/tree/RootNode.h

```diff
@@ -1082,6 +1117,9 @@ inline
 RootNode<ChildT>::RootNode(const RootNode<OtherChildType>& other,
     const ValueType& backgd, TopologyCopy)
     : mBackground(backgd)
+#if OPENVDB_ABI_VERSION_NUMBER >= 10
+    , mOrigin(0, 0, 0)
```

Comment:
This should match the approach you took above - ie set origin to the origin of the other grid, then error if it is non-zero.